lib/ansible/modules/windows/win_firewall_rule.ps1

```diff
@@ -72,9 +72,11 @@ function getFirewallRule ($fwsettings) {
                     if ( $output.$($fwsetting.Key) -ne $fwsettings.$($fwsetting.Key)) {
 
                         if (($fwsetting.Key -eq 'RemoteIP') -and ($output.$($fwsetting.Key) -eq ($fwsettings.$($fwsetting.Key)+'-'+$fwsettings.$($fwsetting.Key)))) {
-                            $donothing=$false
+                            Continue
```

Comment:
I think `Continue` is more readable than setting unused variable. But I can revert to `$donothing=$false` if needed.